src/App.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import Header from './components/Header';
+import TodosList from './components/TodosList';
+import AddTodo from './components/AddTodo';
+import { Box } from '@mui/material';
+import { makeStyles, createStyles } from '@mui/styles';
+
+const useStyles = makeStyles(() =>
+  createStyles({
+    appWrapper: {
+      backgroundColor: '#f8f9fa',
+      height: '100vh'
+    },
+    todoListWrapper: {
+      margin: '0 auto',
+      border: '1px solid black',
+      borderRadius: '5px',
+      minHeight: '50vh'
+    }
+  })
+);
+
+const App: React.FC = () => {
```

Comment:
See [Why is React.FC discouraged?](https://github.com/typescript-cheatsheets/react#function-components). This cheatsheet is really useful for taking tips on how to use TS with React

Reply:
React.FC types have been removed using jscodeshift codemod that was mentioned in the article